### ISSUE TITLE
wochenende_postprocess -a

### ISSUE THREAD

**Issue by vangreuj** [MHH-RCUG/Wochenende#199]
File "csv_to_xlsx_converter.py"

error: No module named 'openpyxl'

Edit: solve with
`conda install -c bioconda openpyxl
`

**growth_rate**

File "main.py"

ModuleNotFoundError: No module named 'lmfit'

pandas.errors.EmptyDataError: No columns to parse from file

**Comment by colindaven** [comment]
Packages are missing, see https://github.com/MHH-RCUG/Wochenende/issues/184

Don't worry about the openpyxl, unless you use the Excel files (you probably won't).

lmfit -  mentioned in Tom's growth rate subfolder:

https://github.com/MHH-RCUG/Wochenende/tree/master/growth_rate

**Comment by colindaven** [comment]
Have you both got the most current version, i.e. by using 
`get_wochenende.sh` ?

Will test.

**Comment by irosenboom** [comment]
Yes, both of us used get_wochenende.sh .

**Comment by colindaven** [comment]
Potential solutions
```
conda deactivate
python3 -m pip install --user lmfit
python3 -m pip install --user lmfit

conda activate wochenende
conda install -c bioconda sambamba openpyxl


#Now start wochenende_postprocess.sh -a from 
#hpc_bc15-07 OR 12 or hpc06 only 
#(where you have done all these installations from) AND
#with activated wochenende conda env
conda activate wochenende
```

**Comment by colindaven** [comment]
I found the bug due to a reordering of activated conda envs in `wochenende_postprocess.sh` (suggested for performance reasons by Lutz). This should be resolved in postprocess v0.34. However, the additional install steps in the comments above should resolve some problems.

**Comment by irosenboom** [comment]
It works fine for me now!
